### PR TITLE
VACMS-16940 DUW v2 FAQ and Additional Info Component

### PR DESCRIPTION
**Pull request by chriskim2311**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary
This PR adds the last component for the review screen which adds FAQs and additional info and links.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16940

## Testing done
Mural for content:
https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1696352911500/80253b1c7200212cbcbcfb940d2cf4a29fed8417?sender=ua4eca0cc168eff677bd03940

## Screenshots
<details><Summary>Accordions same for all answers</Summary>
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
</details>

<details><Summary>Army</Summary>
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
Cursor_and_How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
Cursor_and_How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
</details>

<details><Summary>Navy or Marine Corps</Summary>
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
Cursor_and_How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
</details>

<details><Summary>Air Force</Summary>
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs</details>

<details><Summary>Coast Guard</Summary>
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs</details>

## What areas of the site does it impact?

DUW v2

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

**Comment by chriskim2311** [comment]
@laflannery @thejordanwood This is the last PR for DUW thank you for reviewing!

File: src/applications/discharge-wizard/components/v2/resultsComponents/AdditionalInstructions.jsx [hunk 1]
```diff
@@ -0,0 +1,266 @@
+import React from 'react';
```
Comment (chriskim2311) [inline src/applications/discharge-wizard/components/v2/resultsComponents/AdditionalInstructions.jsx:1]: This is the only new change for this PR. Other changes will resolve when I merge Step 3

**Comment by laflannery** [comment]
@chriskim2311 2 things:
1. Can you change the heading level of "Additional Resources" to and H2? Currently as an H4 its throwing an error:
image

2. For the Air Force flow, I'm not getting what you have in your screen shot but instead I'm getting the results screen with just the action link:
image

**Review by powellkerry — DISMISSED** [review]

**Comment by laflannery** [comment]
LGTM!

**Comment by thejordanwood** [comment]
LGTM

**Review by randimays — COMMENTED** [review]
Found one issue:

I finished testing the Navy/Marine Corps flow and went back to the Review screen to update my answers for the Air Force flow. I figured I probably shouldn't be able to get the below combination of answers (given it says BCNR for Air Force)?

Screenshot 2024-09-20 at 10 53 29 AM

When I went in to edit the BCNR question, I did get the correct choices.

**Comment by chriskim2311** [comment]
Good catch Randi. I have made service branch a "branching" question so the answers will only be valid for the proper service branch answer.

**Review by randimays — APPROVED** [review]
LGTM!

**Pull request merged**